app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(8, 1)) == 8,\
```

Comment:
We don't use `\` when breaking lines as per style convention in our projects - use parenthesis instead:
```
assert (
            sum_instances("Can", "Add") == "CanAdd"
    ), "Sum of 'Can' and 'Add' should be equal to 'CanAdd'"